File: Validus.Console/Validus.Console/Scripts/app/Dtos/TeamDto.js
var ConsoleApp = window.ConsoleApp = window.ConsoleApp || {};

ConsoleApp.Team = function (id, title, defaultMOA, defaultDomicile, quoteExpiryDaysDefault, users, links, relatedCOBs, relatedOffices, allUsers, allLinks, allRelatedCOBs, allRelatedOffices, defaultPolicyType)
{

	this.Id = ko.observable(id);
	this.Title = ko.observable(title);
	this.DefaultMOA = ko.observable(defaultMOA);
	this.DefaultDomicile = ko.observable(defaultDomicile);
	this.QuoteExpiryDaysDefault = ko.observable(quoteExpiryDaysDefault);
	this.DefaultPolicyType = ko.observable(defaultPolicyType);

	this.Users = (users !== undefined && users !== null) ? ko.observableArray(users) : ko.observableArray([]);
	this.Links = (links !== undefined && links !== null) ? ko.observableArray(links) : ko.observableArray([]);  //ko.observableArray(links);
	this.RelatedCOBs = (relatedCOBs !== undefined && relatedCOBs !== null) ? ko.observableArray(relatedCOBs) : ko.observableArray([]); //ko.observableArray(relatedCOBs);
	this.RelatedOffices = (relatedOffices !== undefined && relatedOffices !== null) ? ko.observableArray(relatedOffices) : ko.observableArray([]); //ko.observableArray(relatedOffices);

	this.AllUsers = (allUsers !== undefined && allUsers !== null) ? ko.observableArray(allUsers) : ko.observableArray([]); //ko.observableArray(allUsers);
	this.AllLinks = (allLinks !== undefined && allLinks !== null) ? ko.observableArray(allLinks) : ko.observableArray([]); //ko.observableArray(allLinks);
	this.AllRelatedCOBs = (allRelatedCOBs !== undefined && allRelatedCOBs !== null) ? ko.observableArray(allRelatedCOBs) : ko.observableArray([]); //ko.observableArray(allRelatedCOBs);
	this.AllRelatedOffices = (allRelatedOffices !== undefined && allRelatedOffices !== null) ? ko.observableArray(allRelatedOffices) : ko.observableArray([]); //ko.observableArray(allRelatedOffices);
	
    this.dirtyFlag = new ko.TeamDirtyFlag(this);
};

File: Validus.Console/Validus.Console/DTO/TeamDto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;
using DataAnnotationsExtensions;
using Validus.Models;

namespace Validus.Console.DTO
{
    public class TeamDto
    {
        public Int32 Id { get; set; }

        [Required, StringLength(256)]
        public String Title { get; set; }

        public String SubmissionTypeId { get; set; }

        //public IList<TeamMembershipDto> Memberships { get; set; }

        public IList<BasicUserDto> Users { get; set; }
        public IList<LinkDto> Links { get; set; }
        public IList<COBDto> RelatedCOBs { get; set; }
        public IList<OfficeDto> RelatedOffices { get; set; }

        public IList<BasicUserDto> AllUsers { get; set; }
        public IList<LinkDto> AllLinks { get; set; }
        public IList<COBDto> AllRelatedCOBs { get; set; }
        public IList<OfficeDto> AllRelatedOffices { get; set; }

        [DisplayName("Default MOA"), StringLength(10)]
		public String DefaultMOA { get; set; }

        [DisplayName("Default Domicile"), StringLength(10)]
		public String DefaultDomicile { get; set; }

		[DisplayName("Pricing Actuary")]
        public User PricingActuary { get; set; }

        [DisplayName("Default Policy Type")]
        public String DefaultPolicyType { get; set; }

        //public Int32 PricingActuaryId { get; set; }
        [Integer(ErrorMessage = "Integer value required")]
		[Required, DisplayName("Default Quote Expiry Days *"), Range(1, 365)]
        public Int32 QuoteExpiryDaysDefault { get; set; }

    }
}